crates/rpc/rpc-engine-api/src/engine_api.rs

```diff
@@ -262,12 +262,12 @@ where
         self.inner.task_spawner.spawn_blocking(Box::pin(async move {
             if count > MAX_PAYLOAD_BODIES_LIMIT {
                 tx.send(Err(EngineApiError::PayloadRequestTooLarge { len: count })).ok();
-                return
+                return;
```

Comment:
```suggestion
                return
```

Reply:
Addressed in https://github.com/anton-rs/op-reth/pull/153